service/tooling.gradle

```diff
@@ -18,7 +18,15 @@ idea.module.downloadJavadoc = true
 
 // Immutables has incremental compilation disabled by default
 tasks {
+	boolean isCiServer = System.getenv().containsKey("CI")
+
 	compileJava {
 		options.compilerArgs.add("-Aimmutables.gradle.incremental")
+		// https://broadworkbench.atlassian.net/browse/IA-4442
```

Comment:
I would not link the ticket, but instead would leave a comment to explain that we want the build to fail on CI if the code is not formatted properly. We can always find the original ticket by looking into the git history since you named the branch with the ticket number

Reply:
ok